amplfi/train/models/flow.py

```diff
@@ -133,6 +195,7 @@ def test_step(self, batch, _):
         )
         descaled = self.trainer.datamodule.scale(samples, reverse=True)
         parameters = self.trainer.datamodule.scale(parameters, reverse=True)
+        descaled = self.filter_descaled_parameters(descaled) # filter out samples outside of prior boundaries
```

Comment:
In general, we typically add comments above the relevant code line in this repo. Of course, nothing wrong with adding them inline, but good to keep style consistent